config/locales/en.yml

```diff
@@ -211,12 +211,12 @@ en:
         site:
           attributes:
             location_name:
-              blank: "^Enter a name"
+              blank: "Enter a name"
```

Comment:
Shall we name this more explicitly since there are three fields relating to `name`, maybe `Enter location name` or `Enter site name`?

Reply:
Done